ports/libsimpleservo/jniapi/Cargo.toml

```diff
@@ -14,21 +14,27 @@ test = false
 bench = false
 
 [dependencies]
-android_injected_glue = "0.2"
-android_logger = "0.10"
-gstreamer = { workspace = true }
+android_logger = "0.13"
+# gstreamer = { workspace = true }
 jni = "0.18.0"
 libc = { workspace = true }
 log = { workspace = true }
 serde_json = { workspace = true }
-simpleservo = { path = "../api" }
+
+# TODO: enable native-bluetooth feature for android
+# and remove explicit feature list here
+[dependencies.simpleservo]
+path = "../api"
+default-features = false
+features = ["max_log_level", "webdriver"]
```

Comment:
I think it makes sense to keep using the one line dependency format, which we use pretty consistently in Servo.